pkg/utils/flags.go

```diff
@@ -173,3 +174,38 @@ func NewKlogFlagVal(f *flag.Flag) *KlogFlagVal {
 type boolFlag interface {
 	IsBoolFlag() bool
 }
+
+type DurationVal struct {
```

Comment:
nit: could you add a "docstring" comment for this new exported type, e.g. like

```suggestion
// DurationVal is a wrapper for handling time.Duration as a command line flag
type DurationVal struct {
```